Sprint-2/3-mandatory-implement/3-to-pounds.js

```diff
@@ -4,3 +4,21 @@
 // You will need to declare a function called toPounds with an appropriately named parameter.
 
 // You should call this function a number of times to check it works for different inputs
+
+const toPounds =(penceString)=>{
+     const penceStringWithoutTrailingP = penceString.substring(0, penceString.length - 1);
+  
+    const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0");
+       console.log(paddedPenceNumberString)
```

Comment:
Do we need this `console.log` still here?

Left over console logs can be confusing to people using your code as it clutters the output